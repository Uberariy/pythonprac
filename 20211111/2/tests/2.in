(1,1), (2,2), (11,11)
(1,1), (2,2)
(-1,-2), (3,4), (5,6)
