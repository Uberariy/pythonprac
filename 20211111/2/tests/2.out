Not a triangle
Invalid input
Not a triangle
0.50
